core/src/main/java/bio/terra/pearl/core/service/search/expressions/EnrolleeTermComparisonFacet.java

```diff
@@ -78,28 +78,16 @@ public EnrolleeSearchQueryBuilder generateQueryBuilder(UUID studyEnvId) {
 
         Condition whereCondition;
         if (this.operator.equals(SearchOperators.CONTAINS)) {
-            // If the operator is CONTAINS, we need to wrap the right term in % to make it a valid SQL LIKE clause
```

Comment:
this function started to get very messy so i created a bunch of smaller methods to make it more readable